plugin/src/main/java/io/jenkins/plugins/casc/core/HudsonPrivateSecurityRealmConfigurator.java

```diff
@@ -47,41 +53,114 @@ public HudsonPrivateSecurityRealmConfigurator() {
 
     @CheckForNull
     @Override
-    public CNode describe(HudsonPrivateSecurityRealm instance, ConfigurationContext context) {
+    public CNode describe(HudsonPrivateSecurityRealm instance, ConfigurationContext context)
+        throws Exception {
+        // by default don't export all users
+        if (System.getProperty("io.jenkins.plugins.casc.core.HudsonPrivateSecurityRealmConfigurator.exportUsers", "false").equals("true")) {
```

Comment:
I wasn't sure if we wanted it exported by default, it could be a lot of users, but it's nice to be able to turn it on so you can discover the available properties